packages/flutter_tools/test/application_package_test.dart

```diff
@@ -171,6 +171,7 @@ void main() {
       FileSystem: () => MemoryFileSystem(),
       IOSWorkflow: () => MockIosWorkFlow(),
       Platform: _kNoColorTerminalPlatform,
+      OperatingSystemUtils: () => MockOperatingSystemUtils(),
```

Comment:
I'm surprised you don't need to set up the mock in any way (`when().thenAnswer()`) - what was being called?